tutorial/source/gp.ipynb

```diff
@@ -636,7 +699,10 @@
     "## Reference\n",
     "\n",
     "[1] `Deep Gaussian processes and variational propagation of uncertainty`,<br />&nbsp;&nbsp;&nbsp;&nbsp;\n",
-    "Andreas Damianou"
+    "Andreas Damianou\n",
+    "\n",
+    "[2] `A unifying framework for sparse Gaussian process approximation using power expectation propagation`,<br />&nbsp;&nbsp;&nbsp;&nbsp;\n",
+    "Thang D. Bui, Josiah Yan, and Richard E. Turner"
```

Comment:
can you please add more references?

Reply:
yup :D

Reply:
@martinjankowiak I have added the main references I used for implementations. If there are more, I will add later.